tests/basic/uncontrolled_nondeterminism.rs

```diff
@@ -209,3 +209,21 @@ fn panic_is_yielding() {
         scheduler,
     );
 }
+
+fn iterate_over_hash_set(num_threads: u64) {
```

Comment:
it's not really a `num_threads`, right? `num_entries`?

Reply:
Yes